### ISSUE TITLE
Should ghosts be able to occupy monkeys?

### ISSUE THREAD

**Issue by Probe1** [d3athrow/vgstation13#17904]
I was thinking about that last round so I stuffed a bunch of ghosts into monkeys.  It seemed fun enough for them.

Is this a disastrous idea or not?

**Comment by ShiftyRail** [comment]
Are said monkeys valid ?
Because it's gonna be Dionae nymphs II : zoologic boogaloo, not to mention Monkeys are a resource for many jobs in Medbay and for Xenobio

**Comment by Arthurdentist** [comment]
ahelp: I was trying to do genetics and my monkey started beating me to death with a fire extinguisher.

**Comment by Probe1** [comment]
I see both responses as reasons to do it.  You have to order more monkeys from cargo increasing interaction between the crew.  People are occasionally threatened by NPC tier 5 damage a bite monkeys that can be beaten to death by random things lying around or your fists.

And yeah generally diona nymph rules.  If they're just there on the station without an attempt at becoming crew and shit things up or fight you at all then they're validus saladus.

You could do it as a "once per round" kinda thing so people can't just spam occupy monkey and cause hell.

**Comment by ihadtoregisterforthis** [comment]
Xeno nerf: all the monkeys are trashing the bar and can't be facehugged
Xeno buff: all the monkeys are suddenly self-aware and rushing to xenobio

**Comment by Kammerjunk** [comment]
They took away monkey mode
So we occupied their monkeys

**Comment by Coxswain-Navigator** [comment]
Probably best if you make a chem for it or something so monkeys that are intended to be killed arent trying to escape and killing players in self defense.

**Comment by TurboGay** [comment]
I think it should require player interaction if it was going to be a feature much like how botanists have to plant diona, xenobiologists have to make golem circles, and scientists have to set up borers/pens. I don't think allowing people to just randomly inhabit monkeys is a good idea.

**Comment by Lokiamis** [comment]
There's nothing wrong with just being fucking dead when you die

**Comment by Probe1** [comment]
I only agree in one aspect.  Death should have meaning.  Ways to get back into the round should be give you a worse return than staying alive.

However people dying and not having ways back into the round is very poor game design.  People leave and don't return, leading to worse rounds throughout.  Retaining players in game is an integral issue of a multiplayer game and adding additional paths to rejoin and play isn't wrong.

**Comment by Aurx** [comment]
A: The departments that need monkeys need them alive, so they can do (generally) lethal things to them.  
Letting Grey the Ghost fuck that up for them is just going to piss off players.  There are strong arguments for banning anyone who takes a research monkey and resists being used for their owner's job.  
If this is implemented, it should somehow be limited to monkeys that aren't resources.  

B: Genetics has a lot of monkeys and the ability to turn them into full humans, no strings attached.  This directly contradicts your desired goal of "Ways to get back into the round should be give you a worse return than staying alive."

C: A 5-damage 100-health mob is decently lethal in good hands.  A 2:1 hit ratio isn't that hard to get for a robust player attacking a newcomer.  They also have hands.  

D: As TurboGay mentions, all meaningful forms of rejoining the round on station are living-player initiated. I feel permitting monkeying would severely devalue those.  

E: Dangerously furry.

**Comment by Probe1** [comment]
Being a random chode with no ID or clothes isn't as good as joining the round normally.  Either way I'm getting the feeling this would be fine as a random event but not so much as a regular thing.
>Dangerously furry

Come on.  We can have nice things without having to worry that some degenerate somewhere is a degenerate.

**Comment by Aurx** [comment]
>Come on. We can have nice things without having to worry that some degenerate somewhere is a degenerate.

I agree.  We'd be much better served combating the chronic inability to discern sarcasm in text.

**Comment by Probe1** [comment]
![334](https://user-images.githubusercontent.com/8550416/37807465-f91958a2-2e1b-11e8-9bfe-07cc72545735.gif)

**Comment by Dacendeth** [comment]
Roundstart pun pun when?

**Comment by ghost** [comment]
While I do think it should be a secrets panel button to enable it and there should be a button that infestation event style, silently spawns [mob(s)] somewhere, polls for ghosts for 30s then announces it, it shouldn't be on in general for the same reason as we don't have a mommi spawner aboard before it gets into stealing people's monkeys.

Have the location set to default and hyperlinked so the admin can click to select a specific area
Have a toggle for homogeneous, custom
Have the race default to monkey or whatever and hyperlinked so an admin can set the mob to spawn as a sort of better infestation event
If custom's toggled, a prompt lets them pick the type of mob, amount of that mob, with a box tallying what's up

I vaguely recall the obj system being fairly modular so it shouldn't be too hard to have a warning up saying you're not an antagonist if you check the disclaimer box and you can set general directives for them like establish a home in dorms or whatever.

I thiiink we have a secrets toggle for default AI laws so there's little risk if say, they wanted to as part of an event, make a shuttle, crash it into solars control, spawn 4 AI's with dumb meme laws with a directive to erect shields for an oncoming meteor holocaust in X minutes then hit michael bay and announce it so people go find and interact with them.

Like others said, while we don't want auto-respawn, something to help with I'm ded time to log off could work out and like loadouts, making it easier for an afkmin to mash it and bugger back off helps everyone.

Edit: I guess what I'm trying to say, is another way to do, "tourists are visiting your station" to keep people engaged.

**Comment by Probe1** [comment]
Damn dude I really like the idea of tourists being a second chance role.

**Comment by ghost** [comment]
Yeah, we've had a few neat tourist bus' when we'd go around take photos and shit and generally actually tourist instead of well I've respawned time to "accidentally" grab all access and get back to it.

**Comment by ghost** [comment]
IIRC photo album max photos was enlarged too so the only QoL holdout missing is the loss of the 7x7or whatever size camera photo sizes we used to have

**Comment by abagone** [comment]
So here's my idea what if ghosts where able to possess **specific** simple mobs like Ian, pun pun, mice, pacified slimes or other head pets and pets ordered from cargo.

Mice are already playable and I've had multiple rounds with a clown or someone takes me as a pet and it was pretty fun. 
So why not expand this kind of interaction to all of the pets all heads have one. 
And the bartender I could imagine being more interesting if you had a "_little monkey friend_" to dance on the bar and give people that drinks or be thrown in the **Gibber** if they start acting up!

PS: Also can we make it so that mice stop starving to death please I don't understand why Mice have **hardcore mode enabled**.

**Comment by MiniMeatwad** [comment]
replace resource monkeys with stoks or whatever so whenever turned into the advanced variant through genetics it isn't human so the AI normally doesn't have to listen to it, this also fixes the problem of monkey men being human for silicons

**Comment by abagone** [comment]
MiniMeatwad
>resource monkeys with stoks or **whatever** so whenever  turned into the advanced variant through genetics it **isn't human** so the  AI normally doesn't have to listen to it, this also fixes the problem  of monkey men being human for silicons

Cat beast slaves/experiment test subjects when? A race that is already **explicitly stated in the rules to be killed on site**. They would have even **less rights** than monkeys/dionaeas.

Although should probably consider treating **killing someone's cat beast as equivalent to destroying someone's property** as long as they are contained and or under control like within electroshock pack/chemical implants. 
So people can't just break into genetics and kill all of the cat beasts **without IC repercussions**.

If they want one they'll just have to order their own from cargo you could either order "cat cubes" but have to get the geneticist to make them bigger or buy a big one that comes in a cage and with electroshock pack/chemical implants already equipped!

Something thing we need do is if cats are going to replace monkeys replace the "monkey cubes" with "cat cubes" and make monkeys come in the same quantities of the other small critters for pun pun replacement or if you need a monkey man for brain transplant.

TL;DR:Replace monkeys with cat beasts and able to be possessed by ghosts and add **_cat slavery laws_**.

**Comment by Aurx** [comment]
@abagone
Tajarans should be targets.  Seeing one should always indicate that the hunt is on, not that the geneticist dragged a test subject into the halls.  
Moreover, Tajarans will never not be valid.  Period, full stop.  No matter where they are, no matter what they're doing.

**Comment by abagone** [comment]
Thanks for the criticism @Aurx I was trying to expand on what @MiniMeatwad had mentioned honestly I probably went a little too far with it.

But yeah I feel that there are far too many ways of getting back into the game that are completely dependent on a player in the game initiate it there are only 2 ways that ghost can through their own initiative get back into the round sort of and that is with mice and derelict mommis.

Derelict mommis are pretty much near completely removed any activity mainly because they're meant to just practice build stuff. 

Mice are pretty much effectively visible ghost that can move around/vent crawl, eat food items, squeak and maybe talk to someone with universal knowledge all languages like silicon's and starve to death because for some godforsaken reason they have hardcore mode enabled by default.

You have to wait 5 minutes to respond that either of them.

I would suggest either improving Derelict mommis/Mice or adding new ways to get back into the round. 

My idea to improve the derelict mommis is to give them a law at that makes them potentially hostile to outsiders or at the very least allow them to **fight back Spiders invade your base!** 

My idea to improve mice is to actually make them a nuisance like they should kind of like cockroaches but more intelligent.
Making them more **mobile** by letting them move through unbolted airlocks and **survivable** by making them substantially more resistant to atmospheric changes and no oxygen and finally **remove hardcore mode** from them unless it enabled for everyone else.

In the last thing I would to is allowed them to create a "Mice nests" that can be made with 10 of almost any small random item that that you can find. 
(allow them to pick up small items) 
Stack them on top of each other and and you'll create a spot where players can spawn much quicker as long as it's not destroyed.
This would give another thing for the janitor and assistants to do.

And and add another factor of annoyance to the mice besides eating food by them stealing small items from departments to make nests.

**_My goal with the mice to make them more interesting by making them antagonizes whose goal is to annoy you not kill you._**

**Comment by MiniMeatwad** [comment]
could still be abused, imagine if you hide your emag somewhere and a mouse steals it with their knowledge as a ghost, that would suck

**Comment by Aurx** [comment]
Limiting mice to picking up trash could be neat.  It'd incentivize people to keep the station clean, and ensure that mice wouldn't be stealing things of value.

**Comment by abagone** [comment]
@MiniMeatwad 
But what about how fun it would be to find an emag in a mice nest! 

Let me count the places you could hide your emag where it cannot be stolen by mice in a toilet, a locked personal Locker, a plastic bag with box in space, and maybe even inside the Clowns body cavity! 

But if it's just lying around somewhere or just sitting in a box or plastic bag it's equally as likely to just be found randomly by some assistant and it's not even guaranteed that the ghosts will even see you put it wherever you hide it, 
If you put it in a **bad unsecured hiding spot it's your fault!**

Honestly how often is this edge case going to happen? And if it does then substantial problem people complain about a lot we'll do something about it like not allow mice to pick up emags!

Honestly the good antagonists in this situation what improvise new ways of gaining access to places the bad antagonists will sit there and whine to the admins about the loss of their emag!

**Comment by ghost** [comment]
Policing metagay is reliant on other salty people stalking and noticing bad behavior. It's not reliable to just say well it's the players fault a dead salt mine haunting them abused their knowledge of the round to circumvent however he did hide it. It's hard to see and it's harder to prove. It's not about the emag or anything else they steal, it's about metagaming.

**Comment by MiniMeatwad** [comment]
now that i think about it, allowing mice to pick up small/tiny items like trash to make a nest would be cool

**Comment by Exxion** [comment]
All thumbs are down and last discussion was two months ago. Reopen if people want to discuss it again.
